pgbot/utils/utils.py

```diff
@@ -367,4 +367,41 @@ def format_datetime(dt: Union[int, float, datetime.datetime], tformat: str = "f"
     """
     if isinstance(dt, datetime.datetime):
         dt = dt.replace(tzinfo=datetime.timezone.utc).timestamp()
+
     return f"<t:{int(dt)}:{tformat}>"
+
+
+async def make_message_deletable(
+    *messages: discord.Message,
+    author: Optional[Union[discord.Member, discord.User]] = None,
+    timeout: int = 3600,
+    reaction: Union[str, discord.Emoji] = "❌",
+):
+
+    if not messages:
+        return
+
+    all_messages: List[discord.Message] = list(messages)
```

Comment:
Minor nitpick, `List` is old style type hint, and is deprecated. Throughout the codebase we are using `list` already, and we should be using that. Yes, this is  a 3.9 typehint, but thanks to `from __future__ import annotations`, that will still work on 3.7+ versions